setup.py

```diff
@@ -32,5 +32,6 @@
         "gpt4all",
         "sentence_transformers",
         "docx2txt",
+        "uuid"
```

Comment:
I don't think you need it since python already has `uuid`.

Reply:
Removed this